pgcommitfest/commitfest/templates/workflow.html

```diff
@@ -0,0 +1,244 @@
+  is that the feature the patch introduces is unwanted while, here, the implementation
+  is simply not acceptable.  The workflow takes a different approach and considers
+  both to be less desirable than withdraw.  Considering the distinction between
+  author and committer making the decision to be the key difference the workflow
+  leaves reject as the fallback non-commit option and makes returned a deprecated
+  administrator-only option.
+ </p>
+ <h3>Special PoC Situations</h3>
+ <h4>Moved</h4>
+ <p>
+  Moved PoCs are a side-effect of having commitfests, and volume.  Many active
+  patches cannot be gotten to within a commitfest.  In order to keep them visible
+  they must be moved to another commitfest, creating a new PoC with the same
+  state.  The PoC they were moved from is updated to "Moved" to indicate this
+  flow-of-time action.
+ </p>
+ <h4>Is Abandoned</h4>
+ <p>
+  This check returns true if the PoC state is in the set of active states
+  but the commitfest is closed.  Conceptually, this is the same as
+  withdrawn, but through inaction.
+ </p>
+ <h3>Reverted Patches</h3>
+ <p>
+  Not every patch that is committed stays that way.  The Workflow doesn't have
+  any statuses for this, though the user interface does provide an action that
+  a committer can invoke.  Upon doing so the PoC status is updated to
+  author.  The history flow of commited followed by author indictes a probable
+  revert.
+ </p>
+ <h3>Patches</h3>
+ <h4>Overview</h4>
+ <p>
+  Patches are the basic unit of work in the workflow.  They are moved around
+  between various commitfest bins during their lifetime to reflect how they
+  interact with the focus of the community during each time period.
+  This movement, combined with the fact that a patch may only have a single
+  status, means that only one of the patch's PoCs can have a state that isn't
+  "Moved".  While a patch's status is derived from its primary PoC, everything
+  else is attached to the patch itself.
+ </p>
+ <h4>Authors, Reviewers, and Committers</h4>
+ <p>
+  Especially in open source projects, attribution for work is important.  Git
+  commit messages include author and reviewer attributions (among others) and
+  inherently identify the committer.  To aid the committer in properly recording
+  attributions in the commit message record authors and reviewers on the patch.
+ </p>
+ <p>
+  Additionally, the commitfest application uses this information to provide
+  user-specific reporting and notifications.
+ </p>
+ <h4>Threads</h4>
+ <p>
+  The actual content of a patch does not existing within the commitfest application
+  (more-or-less); instead, the patch file submission and review process is all done
+  on the -hackers mailing list.  Threads are the means by which the commitfest
+  is made aware of these files and can incorporate information pertaining to them
+  into its reporting.
+ </p>
+ <h3>Commitfests</h3>
```

Comment:
You already sortof introduced commitfests in the workflow section. I think this needs some re-ordering/deduplicating/merging of information.

Reply:
As with patch, its intended more as a reference section for the topic.  But I too don't consider it quite publication worthy yet.  Just waiting to see if new/different material needs to be considered as well.